lib/src/install.rs

```diff
@@ -644,8 +644,16 @@ async fn initialize_ostree_root_from_self(
     }
     f.flush()?;
 
+    let fstab_path = rootfs.join("etc/fstab");
+    state.lsm_label(&fstab_path, "/etc/fstab".into(), false)?;
+
     if let Some(contents) = state.root_ssh_authorized_keys.as_deref() {
-        osconfig::inject_root_ssh_authorized_keys(&root, contents)?;
+        let tmpf_path = osconfig::inject_root_ssh_authorized_keys(&root, contents)?;
+        state.lsm_label(
+            &rootfs.join(tmpf_path.clone()),
```

Comment:
This is a bit silly (sorry!) I think:
```rust
-            &rootfs.join(tmpf_path.clone()),
+            &rootfs.join(&tmpf_path),
```   
better (so, rusty in rust).

But the tests fails right now with:
```
DEBUG exec: "matchpathcon" "-n" "/etc/fstab"
DEBUG Label for /target/etc/fstab is system_u:object_r:etc_t:s0
DEBUG exec: "chcon" "-h" "-h" "system_u:object_r:etc_t:s0" "/target/etc/fstab"
Injected: etc/tmpfiles.d/bootc-root-ssh.conf
DEBUG exec: "matchpathcon" "-n" "/etc/tmpfiles.d/bootc-root-ssh.conf"
DEBUG Label for /target/etc/tmpfiles.d/bootc-root-ssh.conf is system_u:object_r:etc_t:s0
DEBUG exec: "chcon" "-h" "-h" "system_u:object_r:etc_t:s0" "/target/etc/tmpfiles.d/bootc-root-ssh.conf"
chcon: cannot access '/target/etc/tmpfiles.d/bootc-root-ssh.conf': No such file or directory
ERROR Installing to filesystem: Creating ostree deployment: Labeling /etc/tmpfiles.d/bootc-root-ssh.conf: Task chcon failed: ExitStatus(unix_wait_status(256))
```
which I need to figure out still (it almost identical to the fstab call above which is puzzling)

Reply:
I think you want to create the labels relative to `root` and not `rootfs`, since `root` is pointing into the ostree deployment (similar to immediately above with `root.open_with("etc/fstab", ...)`)

Reply:
Something along the lines of...
```
-    let fstab_path = rootfs.join("etc/fstab");
+    let fstab_path = Utf8PathBuf::try_from(root.canonicalize("etc/fstab")?)?;
```
and then similar below for labeling the ssh keys